lib/interactive_viewer.ml

```diff
@@ -149,9 +156,19 @@ let view (patches : Patch.t list) =
           operation_info z_patches;
           change_summary z_patches;
           current_operation z_patches;
-          W.vscroll_area
-            ~state:(Lwd.get curr_scroll_state)
-            ~change:change_scroll_state
+          (* W.vscroll_area
+               ~state:(Lwd.get curr_scroll_state)
+               ~change:change_scroll_state
+             @@ W.hscroll_area
+                  ~state:(Lwd.get curr_scroll_state_h)
+                  ~change:change_scroll_state_h
+             @@ current_hunks z_patches; *)
+          W.hscroll_area
+            ~state:(Lwd.get curr_scroll_state_h)
+            ~change:change_scroll_state_h
+          @@ W.vscroll_area
+               ~state:(Lwd.get curr_scroll_state)
+               ~change:change_scroll_state
           @@ current_hunks z_patches;
           Lwd.pure
           @@ Ui.keyboard_area
```

Comment:
> What are your thoughts on incorporating horizontal scroll keys v and b?

Key bindings would be a great addition ! In what other tools are `v` and `b` used for scrolling ? As we plan to have `j` and `k` for vertical scrolling, I'd suggest `h` and `l` to be consistent in the vim-like keybindings.

Reply:
Hi @Julow, I understand that we currently use h for opening the help panel. we may need to explore alternative options for incorporating horizontal scrolling keys while ensuring consistency and usability in our application. What are your thoughts on this?

Reply:
Good catch! Then `v` and `b` seem like good options.